odl_video/envs_test.py

```diff
@@ -148,3 +150,13 @@ def test_get_key():
             b'/q0txR0v1rqmowS1mQIDAQAB\n'
             b'-----END PUBLIC KEY-----\n'
         )
+
+
+def test_parse_env():
+    testpath = 'testenv.txt'
+    with open(testpath, 'w') as testfile:
+        testfile.write('FOO_VAR=bar_var\nFOO_NUM=42')
```

Comment:
Can you change one of the lines to include an `"export "`, and have one of the values include its own `=` sign?